### ISSUE TITLE
Error while building Conductor Server and Conductor UI docker image

### ISSUE THREAD

**Issue by JCHacking** [Netflix/conductor#3632]
**Describe the bug**
When I try to create a Conductor Server and Conductor UI image by referencing the existing Dockerfile it fails.

Conductor Server Output: **(Although the branch indicates that it is version 3.13.6, I see in the output that it says version 3.14.0-SNAPSHOT.)**
```bash
...
#0 42.95 > Configure project :
#0 42.95 Inferred project: conductor, version: 3.14.0-SNAPSHOT
...
#0 185.3
#0 185.3 FAILURE: Build failed with an exception.
#0 185.3
#0 185.3 * What went wrong:
#0 185.3 Execution failed for task ':conductor-client:spotlessJavaCheck'.
#0 185.3 > The following files had format violations:
#0 185.3       src/main/java/com/netflix/conductor/client/http/EventClient.java
#0 185.3           @@ -30,6 +30,7 @@
#0 185.3            public·class·EventClient·extends·ClientBase·{
#0 185.3            ····private·static·final·GenericType<List<EventHandler>>·eventHandlerList·=
#0 185.3            ············new·GenericType<List<EventHandler>>()·{};
#0 185.3           +
#0 185.3            ····/**·Creates·a·default·metadata·client·*/
#0 185.3            ····public·EventClient()·{
#0 185.3            ········this(new·DefaultClientConfig(),·new·DefaultConductorClientConfiguration(),·null);
#0 185.3   Run './gradlew :conductor-client:spotlessApply' to fix these violations.
#0 185.3
#0 185.3 * Try:
#0 185.3 > Run with --info or --debug option to get more log output.
#0 185.3
#0 185.3 * Exception is:
#0 185.3 org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':conductor-client:spotlessJavaCheck'.
#0 185.3        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:142)
#0 185.3        at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:282)
#0 185.3        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:140)
#0 185.3        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:128)
#0 185.3        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
#0 185.3        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
#0 185.3        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
#0 185.3        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
#0 185.3        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
#0 185.3        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
#0 185.3        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
#0 185.3        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
#0 185.3        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
#0 185.3        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
#0 185.3        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
#0 185.3        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:69)
#0 185.3        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:327)
#0 185.3        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:314)
#0 185.3        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:307)
#0 185.3        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:293)
#0 185.3        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:420)
#0 185.3        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:342)
#0 185.3        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
#0 185.3        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
#0 185.4 Caused by: org.gradle.api.GradleException: The following files had format violations:
#0 185.4     src/main/java/com/netflix/conductor/client/http/EventClient.java
#0 185.4         @@ -30,6 +30,7 @@
#0 185.4          public·class·EventClient·extends·ClientBase·{
#0 185.4          ····private·static·final·GenericType<List<EventHandler>>·eventHandlerList·=
#0 185.4          ············new·GenericType<List<EventHandler>>()·{};
#0 185.4         +
#0 185.4          ····/**·Creates·a·default·metadata·client·*/
#0 185.4          ····public·EventClient()·{
#0 185.4          ········this(new·DefaultClientConfig(),·new·DefaultConductorClientConfiguration(),·null);
#0 185.4 Run './gradlew :conductor-client:spotlessApply' to fix these violations.
#0 185.4        at com.diffplug.gradle.spotless.SpotlessCheck.performAction(SpotlessCheck.java:111)
#0 185.4        at com.diffplug.gradle.spotless.SpotlessCheck.performAction(SpotlessCheck.java:53)
#0 185.4        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
#0 185.4        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
#0 185.4        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
#0 185.4        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:125)
#0 185.4        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:58)
#0 185.4        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:51)
#0 185.4        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:29)
#0 185.4        at org.gradle.api.internal.tasks.execution.TaskExecution$3.run(TaskExecution.java:236)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68)
#0 185.4        at org.gradle.api.internal.tasks.execution.TaskExecution.executeAction(TaskExecution.java:221)
#0 185.4        at org.gradle.api.internal.tasks.execution.TaskExecution.executeActions(TaskExecution.java:204)
#0 185.4        at org.gradle.api.internal.tasks.execution.TaskExecution.executeWithPreviousOutputFiles(TaskExecution.java:187)
#0 185.4        at org.gradle.api.internal.tasks.execution.TaskExecution.execute(TaskExecution.java:165)
#0 185.4        at org.gradle.internal.execution.steps.ExecuteStep.executeInternal(ExecuteStep.java:89)
#0 185.4        at org.gradle.internal.execution.steps.ExecuteStep.access$000(ExecuteStep.java:40)
#0 185.4        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:53)
#0 185.4        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:50)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
#0 185.4        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:50)
#0 185.4        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:40)
#0 185.4        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:68)
#0 185.4        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:38)
#0 185.4        at org.gradle.internal.execution.steps.CancelExecutionStep.execute(CancelExecutionStep.java:41)
#0 185.4        at org.gradle.internal.execution.steps.TimeoutStep.executeWithoutTimeout(TimeoutStep.java:74)
#0 185.4        at org.gradle.internal.execution.steps.TimeoutStep.execute(TimeoutStep.java:55)
#0 185.4        at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:51)
#0 185.4        at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:29)
#0 185.4        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.executeDelegateBroadcastingChanges(CaptureStateAfterExecutionStep.java:124)
#0 185.4        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.execute(CaptureStateAfterExecutionStep.java:80)
#0 185.4        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.execute(CaptureStateAfterExecutionStep.java:58)
#0 185.4        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:48)
#0 185.4        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:36)
#0 185.4        at org.gradle.internal.execution.steps.BuildCacheStep.executeWithoutCache(BuildCacheStep.java:181)
#0 185.4        at org.gradle.internal.execution.steps.BuildCacheStep.lambda$execute$1(BuildCacheStep.java:71)
#0 185.4        at org.gradle.internal.Either$Right.fold(Either.java:175)
#0 185.4        at org.gradle.internal.execution.caching.CachingState.fold(CachingState.java:59)
#0 185.4        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:69)
#0 185.4        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:47)
#0 185.4        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:36)
#0 185.4        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:25)
#0 185.4        at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:36)
#0 185.4        at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:22)
#0 185.4        at org.gradle.internal.execution.steps.SkipUpToDateStep.executeBecause(SkipUpToDateStep.java:110)
#0 185.4        at org.gradle.internal.execution.steps.SkipUpToDateStep.lambda$execute$2(SkipUpToDateStep.java:56)
#0 185.4        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:56)
#0 185.4        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:38)
#0 185.4        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:73)
#0 185.4        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:44)
#0 185.4        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:37)
#0 185.4        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:27)
#0 185.4        at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:89)
#0 185.4        at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:50)
#0 185.4        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:114)
#0 185.4        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:57)
#0 185.4        at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:76)
#0 185.4        at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:50)
#0 185.4        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.executeWithNoEmptySources(SkipEmptyWorkStep.java:254)
#0 185.4        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.execute(SkipEmptyWorkStep.java:91)
#0 185.4        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.execute(SkipEmptyWorkStep.java:56)
#0 185.4        at org.gradle.internal.execution.steps.RemoveUntrackedExecutionStateStep.execute(RemoveUntrackedExecutionStateStep.java:32)
#0 185.4        at org.gradle.internal.execution.steps.RemoveUntrackedExecutionStateStep.execute(RemoveUntrackedExecutionStateStep.java:21)
#0 185.4        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsStartedStep.execute(MarkSnapshottingInputsStartedStep.java:38)
#0 185.4        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:43)
#0 185.4        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:31)
#0 185.4        at org.gradle.internal.execution.steps.AssignWorkspaceStep.lambda$execute$0(AssignWorkspaceStep.java:40)
#0 185.4        at org.gradle.api.internal.tasks.execution.TaskExecution$4.withWorkspace(TaskExecution.java:281)
#0 185.4        at org.gradle.internal.execution.steps.AssignWorkspaceStep.execute(AssignWorkspaceStep.java:40)
#0 185.4        at org.gradle.internal.execution.steps.AssignWorkspaceStep.execute(AssignWorkspaceStep.java:30)
#0 185.4        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:37)
#0 185.4        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:27)
#0 185.4        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:44)
#0 185.4        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:33)
#0 185.4        at org.gradle.internal.execution.impl.DefaultExecutionEngine$1.execute(DefaultExecutionEngine.java:76)
#0 185.4        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:139)
#0 185.4        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:128)
#0 185.4        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
#0 185.4        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
#0 185.4        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
#0 185.4        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
#0 185.4        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
#0 185.4        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
#0 185.4        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
#0 185.4        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
#0 185.4        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
#0 185.4        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
#0 185.4        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
#0 185.4        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:69)
#0 185.4        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:327)
#0 185.4        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:314)
#0 185.4        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:307)
#0 185.4        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:293)
#0 185.4        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:420)
#0 185.4        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:342)
#0 185.4        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
#0 185.4        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
#0 185.4
#0 185.4
#0 185.4 * Get more help at https://help.gradle.org
#0 185.4
#0 185.4 Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
#0 185.4
#0 185.4 You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
#0 185.4
#0 185.4 See https://docs.gradle.org/7.5.1/userguide/command_line_interface.html#sec:command_line_warnings
#0 185.4
#0 185.4 Execution optimizations have been disabled for 2 invalid unit(s) of work during this build to ensure correctness.
#0 185.4 Please consult deprecation warnings for more details.
#0 185.4 193 actionable tasks: 182 executed, 11 up-to-date
#0 185.4
#0 185.4 BUILD FAILED in 3m 4s
#0 185.5
#0 185.5 Publishing build scan...
#0 186.1 https://gradle.com/s/wl4ztsilo7yw4
#0 186.1
------
Dockerfile:17
--------------------
  15 |
  16 |     # Build the server on run
  17 | >>> RUN ./gradlew build -x test --stacktrace
  18 |
  19 |     # ===========================================================================================================
--------------------
ERROR: failed to solve: process "/bin/sh -c ./gradlew build -x test --stacktrace" did not complete successfully: exit code: 1
```

Conductor UI Output:
```bash
...
#0 102.3 error workbox-webpack-plugin@6.6.1: The engine "node" is incompatible with this module. Expected version ">=16.0.0". Got "14.21.3"
#0 102.3 error Found incompatible module.
#0 102.3 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
------
Dockerfile:19
--------------------
  17 |
  18 |     # Installs all node packages. Cached unless package.json changes
  19 | >>> RUN yarn install
  20 |
  21 |     # Copies everything else over to Docker environment
--------------------
ERROR: failed to solve: process "/bin/sh -c yarn install" did not complete successfully: exit code: 1

```

**Details**
Conductor version: 3.13.6

**To Reproduce**
1. Clone conductor repository (Tag 3.13.6)
2. Make build from conductor/docker/server/Dockerfile and /conductor/docker/ui/Dockerfile
3. See error

**Expected behavior**
Troubleshoot problems so that Conductor Server and Conductor UI builds can be performed.

**Comment by justinjeffery-ipf** [comment]
bump

**Comment by GabrielOctavianPopa** [comment]
bump

**Comment by ashwineaso** [comment]
bump

**Comment by goneneren** [comment]
Have the same error

**Comment by alberttwong** [comment]
I also have the same issue.

**Comment by alberttwong** [comment]
answer at https://github.com/Netflix/conductor/issues/3674

**Comment by JCHacking** [comment]
Solved with the following Merge: #3680 3680

**Comment by justinjeffery-ipf** [comment]
@JCHacking Doesn't appear solved...

```
C:\Code\_INTEGRATIONS\conductor\docker>docker-compose build
[+] Building 124.6s (12/16)
[+] Building 124.8s (12/16)
 => => sha256:9d19ee268e0d7bcf6716e6658ee1b0384a71d6f2f9aa1ae2085610cf7c7b316f 30.43MB / 30.43MB                                                                                                          3.6s
 => => sha256:1acd9f0b851bbe1a40c9208d7652c706c94432c0ae0e38d03432f5e8dc0f6959 198.55MB / 198.55MB                                                                                                       12.8s
 => => sha256:715f805aa7a77fd5cc323e76c291037ce343bc2378976256f12d080d0b1ef699 175B / 175B                                                                                                                3.8s
 => => extracting sha256:9d19ee268e0d7bcf6716e6658ee1b0384a71d6f2f9aa1ae2085610cf7c7b316f                                                                                                                 6.0s
 => => extracting sha256:32db0ad82863deaedd2f9a365e52f595085778d28bd1e6f1d351107590cad806                                                                                                                 5.6s
 => => extracting sha256:1acd9f0b851bbe1a40c9208d7652c706c94432c0ae0e38d03432f5e8dc0f6959                                                                                                                 7.5s
 => => extracting sha256:715f805aa7a77fd5cc323e76c291037ce343bc2378976256f12d080d0b1ef699                                                                                                                 0.0s
 => [conductor-server stage-1 2/6] RUN mkdir -p /app/config /app/logs /app/libs                                                                                                                          10.2s 
 => [conductor-server builder 2/4] COPY . /conductor                                                                                                                                                      1.6s 
 => [conductor-server builder 3/4] WORKDIR /conductor                                                                                                                                                     0.1s 
 => ERROR [conductor-server builder 4/4] RUN ./gradlew build -x test --stacktrace           
```
